website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,275 @@
+
+In our example [`pkg/controller/memcached/memcached_controller.go`][memcached_controller] is copied to `controllers/memcached_controller.go`.
+
+Particularly, note the following changes:
+
+1. The naming convention of the reconciler struct may differ. For example, in case of memcached operator the reconciler struct in SDK project is mentioned as [`ReconcileMemcached`][sdk_reconcile_struct] whereas in Kubebuilder project it would be `MemcachedReconciler`.
+
+```go
+type MemcachedReconciler struct {
+	Log    logr.Logger
+	Client client.Client
+	Scheme *runtime.Scheme
+}
+```
+
+2. Update the import aliases in `Reconcile` logic and copy over the remaining related functions from `pkg/controller/<kind>/<kind>_controller.go` to `api/<version>/<kind>_controller.go`.
+
+Refer to [`memcached_controller.go`][kb_memcached_controller] for the example implementation of `reconcile` logic which would be present in `api/v1alpha1/memcached_controller.go` of Memcached operator.
+
+## Generate CRDs
+
+Run [`make manifests`][generate_crd] to generate CRD manifests. They would be generated inside the `config/` folder.
+
+```sh
+$ make manifests
+go: creating new go.mod: module tmp
+go: finding sigs.k8s.io v0.2.5
+go: finding sigs.k8s.io/controller-tools/cmd v0.2.5
+go: finding sigs.k8s.io/controller-tools/cmd/controller-gen v0.2.5
+/Users/admin/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+```
+
+## Operator Manifests
+
+### Operator deployment manifests
+
+The operator deployment manifest [`operator.yaml`][deployment_yaml] present inside `deploy/` folder in SDK project can be copied to `config/manager/manager.yaml`
+
+**Note**: 
+The kustomize file requires the operator deployment manifest to have the field `namespace` which is missing in the current operator.yaml of sdk project.
+
+### RBAC permissions
+
+The RBAC manifests present in `config/rbac/` of a kubebuilder project are generated through the [RBAC markers][rbac_markers] present in `controller/<kind>_controller.go`. They can be added as comments above the `Reconcile` method.
+
+In our example of memcached operator, add the following rbac markers in `memcached_controller.go`:
+
+```Go
+
+...
+// +kubebuilder:rbac:groups=apps,resources=deployments;pods;daemonsets;replicasets;statefulsets,verbs=get;update;patch;list;create;delete;watch
+func (r *MemcachedReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	_ = context.Background()
+	reqLogger := log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
+	reqLogger.Info("Reconciling Memcached")
+
+...
+```
+**Note**:
+To update `config/rbac/role.yaml` after changing the markers, run `make manifests`.
+
+## Build and run the operator:
```

Comment:
@joelanford @varshaprasad96  
I'm thinking for the bild and run part of the migration guide, there's no actual migration steps involved.
Since it's just about ensuring that you can build and run your operator, can we just link to the [`Ruild and run the operator`](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/kubebuilder/quickstart.md#build-and-run-the-operator) section in the user-guide guide rather than repeating what's already there.